app/lib/database/model/Subscription.js

```diff
@@ -110,6 +108,8 @@ export default class Subscription extends Model {
 
 	@json('tags', sanitizer) tags;
 
+	@field('join_code_required') joinCodeRequired;
+
```

Comment:
Looks like it's not a necessary change, since it already exists on the model 😃  just undo and all fine